### PR TITLE
[Slideshow] Add ambient movement to background

### LINKED ISSUE #2341: [Slideshow] Add ambient movement to background
## Summary

Add ambient animation to the Slideshow section, as prototyped in #2229. 

## Details

This animation will be off by default. It should be implemented as a setting in a new dropdown, like so:
 
Screenshot 2023-02-28 at 11 45 22 AM

### PR DESCRIPTION
**Pull request by kjellr**
### PR Summary: 

Adds an option for ambient animation to the Slideshow section's background images.

https://user-images.githubusercontent.com/1202812/224727458-4fa51b8f-11d1-4d9a-8f39-81dccf4bcfa6.mp4

### Why are these changes introduced?

Closes #2341. 

### What approach did you take?

Uses the same approach (and existing CSS) that was added in https://github.com/Shopify/dawn/pull/2342.

This implements a new dropdown menu [as shown in the ticket](https://github.com/Shopify/dawn/pull/2341). When "Ambient movement" is selected, a CSS class is applied to the media elements within the Image Banner, triggering a long-duration CSS animation that slowly moves the image around within its frame. As this feature is purely aesthetic, it supports the visitor's prefers-reduced-motion preference.

Like https://github.com/Shopify/dawn/pull/2342, it also conditionally loads in a slightly higher-res image when the ambient animation is active, in order to account for the scale effect that's applied to the image. 

### Visual impact on existing themes

This feature is off by default. When it's turned on, all background images in the slideshow will animate ambiently. 

### Testing steps/scenarios

1. [Visit the demo store](https://os2-demo.myshopify.com/admin/themes/139517362198/editor).
2. Add and select a Slideshow.
3. Look for the new "Image behavior" dropdown underneath the new "Animations" header in the Slideshow's sidebar.
4. Select "Ambient movement" from that dropdown.
5. Add image backgrounds to individual slides, verify that they appear as expected.
6. Try various breakpoints, verify that they appear as expected.

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: locales/en.default.schema.json [hunk 1]
```diff
@@ -2953,6 +2953,18 @@
...
+        },
+        "image_behavior": {
+          "options__1": {
+            "label": "None"
+          },
+          "options__2": {
+            "label": "Ambient movement"
+          },
+          "label": "Image behavior"
+        },
```
[the remark this answers is not in the document]
Comment (kjellr) [reply locales/en.default.schema.json:2967]: I've updated https://github.com/Shopify/dawn/pull/2385 to use new shared `sections.all.animation` strings. Let's focus on merging that PR first, and then I'll set this one to point to those strings too. 👍

File: sections/slideshow.liquid [hunk 2]
```diff
@@ -338,6 +347,26 @@
       "label": "t:sections.slideshow.settings.change_slides_speed.label",
       "default": 5
     },
+    {
+      "type": "header",
+      "content": "t:sections.slideshow.settings.animations.content"
```
Comment (LucasLacerdaUX) [inline sections/slideshow.liquid:352]: This is pointing to an invalid translation:
image

I think you meant 
```suggestion
      "content": "t:sections.all.animation.content"
```

**Review by LucasLacerdaUX — COMMENTED** [review]
Just a little typo on the translations

**Review by LucasLacerdaUX — APPROVED** [review]
Looks good to me! Aligned with Stu's comment above - the extra loading time seems to be happening more often when the animation is turned on because the image is larger and we're using a high-res asset on our test store.

Not only I was unable to reproduce it consistently, but I also got similar issues a couple times when the ambient movement animation was turned off. I'm pretty confident this isn't caused by this PR - it just becomes more noticeable in the test scenario we have due to larger images.

**Review by metamoni — APPROVED** [review]

**Pull request merged**